### PR TITLE
Slogan showdown

### PR DESCRIPTION
**Pull request by ajaydono**

### CODE REVIEW

File: _posts/2024-03-06-firo-slogan-showdown.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+As part of the event, an award will be given to the participant who comes up with the most creative and meaningful slogan. If you're curious about what the award might be, we welcome you to join our event filled with surprises!
+
+## Event Details
+
+* **Start Date**: 6th March, 2024 
+* **Deadline for Submission**: 20th March 2024 
+
+### **How to Participate** 
+
+You can submit your entry with the hashtag **#FiroSloganShowdown** in the official [Firo Telegram Channel](https://t.me/firoorg), or reply to the Twitter/X post.
```
Comment (erciccione) [inline _posts/2024-03-06-firo-slogan-showdown.md:24]: We could also mention the matrix room since it's bridged with telegram

**Review by erciccione — COMMENTED** [review]
I suggest to use the banner below. It's the same as the one suggested in the pr, so no degraded quality, but 63% lighter (226kb). It will increase performance.

banner

**Review by ajaydono — COMMENTED** [review]
Done changes

File: _posts/2024-03-06-firo-slogan-showdown.md (hunk 1 above)
Comment (ajaydono) [reply _posts/2024-03-06-firo-slogan-showdown.md]: Sure

**Review by erciccione — APPROVED** [review]

**Pull request merged**